packages/main/src/StandardListItem.hbs

```diff
@@ -33,12 +33,12 @@
 
 {{#*inline "iconBegin"}}
 	{{#if displayIconBegin}}
-		<ui5-icon part="icon" name="{{icon}}" class="ui5-li-icon"></ui5-icon>
+		<ui5-icon part="icon" name="{{icon}}" class="ui5-li-icon" accessible-role="presentation" aria-hidden="true"></ui5-icon>
```

Comment:
@dobrinyonkov we discussed these changes couple of weeks ago, but please review them once again so we can finalise the change.

Reply:
Yes, these and all other List related changes are OK for me.

Reply:
Thank you : )

Reply:
As far as I know, all List-related changes by my side are OK.